src/components/navigation/Navigation.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Navigation.css';
+import {NavLink} from 'react-router-dom';
+
+function Navigation(props) {
+
+    return (
+        <>
+           <nav>
+
```

Comment:
misschien nog het logo toevoegen. die staan eigenlijk altijd op elke website wel in de nav